tests/src/JIT/Regression/JitBlue/GitHub_1323/GitHub_1323.cs

```diff
@@ -0,0 +1,22 @@
+using System;
```

Comment:
Is @mikedn a microsoft employee?  I thought he was external.  He should add a MIT license (c) to him.